addons/arrays/test_shuffle.sqf

```diff
@@ -21,18 +21,14 @@ TEST_OP(count _result,==,count _original,_fn);
     TEST_OP(_x,in,_original,_fn);
 } forEach _result;
 
-// Test depecated version.
+// test modify original
```

Comment:
Refactorise the old test to do what it did before and _add_ a new one that checks if the original array is modified.